internal-dependencies/pom.xml

```diff
@@ -138,11 +138,44 @@
         <artifactId>openjpa</artifactId>
         <version>${version.openjpa}</version>
       </dependency>
+
+      <!-- groovy script engine dependencies -->
+      <dependency>
+        <groupId>org.codehaus.groovy</groupId>
+        <artifactId>groovy</artifactId>
+        <version>${version.groovy}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.codehaus.groovy</groupId>
+        <artifactId>groovy-jsr223</artifactId>
+        <version>${version.groovy}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.codehaus.groovy</groupId>
+        <artifactId>groovy-json</artifactId>
+        <version>${version.groovy}</version>
+      </dependency>
       <dependency>
         <groupId>org.codehaus.groovy</groupId>
-        <artifactId>groovy-all</artifactId>
+        <artifactId>groovy-xml</artifactId>
         <version>${version.groovy}</version>
       </dependency>
+      <dependency>
+        <groupId>org.codehaus.groovy</groupId>
+        <artifactId>groovy-templates</artifactId>
+        <version>${version.groovy}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.codehaus.groovy</groupId>
+        <artifactId>groovy-dateutil</artifactId>
+        <version>${version.groovy}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.codehaus.groovy</groupId>
+        <artifactId>groovy-datetime</artifactId>
+        <version>${version.groovy}</version>
+      </dependency>
+
```

Comment:
You can import the Groovy BOM so you don't need to track all versions separately.

```suggestion
      <!-- groovy script engine dependencies -->
      <dependency>
        <groupId>org.codehaus.groovy</groupId>
        <artifactId>groovy-all</artifactId>
        <version>${version.groovy}</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
```

Reply:
I actually tried this first, but for some reason, Maven didn't make the Groovy dependencies available to the child poms. I was getting a "missing version" error. I'll try again, and ping you if I run into some problems.

Reply:
The solution was to use `groovy-bom` instead of `groovy-all` since the latter is not a BOM.

Reply:
Sorry for the mix-up 😄👍